spring-integration-core/src/main/java/org/springframework/integration/handler/ExpressionEvaluatingMessageHandler.java

```diff
@@ -37,8 +38,8 @@
 
 	public ExpressionEvaluatingMessageHandler(Expression expression) {
 		Assert.notNull(expression, "Expression must not be null");
```

Comment:
The `Expression` in the message looks odd. 
Should be the param name.

Will fix on merge.